include/mbedtls/cipher.h

```diff
@@ -599,10 +619,12 @@ int mbedtls_cipher_reset( mbedtls_cipher_context_t *ctx );
 /**
  * \brief               This function adds additional data for AEAD ciphers.
  *                      Currently supported with GCM and ChaCha20+Poly1305.
- *                      Must be called exactly once, after mbedtls_cipher_reset().
+ *                      This must be called exactly once, after
+ *                      mbedtls_cipher_reset().
  *
- * \param ctx           The generic cipher context.
- * \param ad            The additional data to use.
+ * \param ctx           The generic cipher context. This must be initialized.
+ * \param ad            The additional data to use. If `ad_len > 0`, then this
+ *                      must be a readable buffer of at least \p ad_len Bytes.
  * \param ad_len        the Length of \p ad.
```

Comment:
Pre-existing: Correct spelling and be more precise, e.g. "The length of \p ad in Bytes."